src/textual_dev/previews/colors.py

```diff
@@ -56,15 +76,17 @@ class ColorsApp(App[None]):
     BINDINGS = [("d", "toggle_dark", "Toggle dark mode")]
 
     def compose(self) -> ComposeResult:
-        yield Content(ColorButtons())
         yield Footer()
+        with ColorTabs("Theme Colors", "Named Colors"):
+            yield Content(ThemeColorButtons())
+            yield Vertical(NamedColorsView())
```

Comment:
Do we need the outer containers?

Reply:
The first `Content` is used to dynamically mount `ColorsView` in  `update_view`, I guess to "toggle dark mode" to work.

The second `Vertical` is needed so that the tab content to be displayed at all. I have zero idea, why though...